src/test/app/claims/ccjModelConverter.ts

```diff
@@ -0,0 +1,141 @@
+ } from 'test/data/entity/responseData'
+// Claimant-Response Draft and Data
+import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
+import { claimantResponseDraftWithPaymentMethodAccepted,
+  claimantResponseDraftWithPaymentMethodRejectedAndImmediatePaymentProposed,
+  claimantResponseDraftWithPaymentMethodRejectedAndPaymentBySetDateProposed
+} from 'test/data/draft/claimantResponseDraft'
+// CCJ Draft and Data
+// import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
+import {
+  ccjIssueRequestPayImmediately,
+  ccjIssueRequestPayBySetDate,
+  ccjIssueRequestPayByInstalments,
+  ccjIssueRequestPayImmediatelyForCompany,
+  ccjIssueRequestPayBySetDateForCompany,
+  ccjIssueRequestPayByInstalmentsForCompany
+} from 'test/data/entity/ccjData'
+import { Claim } from 'claims/models/claim'
+import { Response } from 'claims/models/response'
+import { Draft } from '@hmcts/draft-store-client'
+// import { Defendant } from 'drafts/models/defendant'
+
+function prepareClaim (responseTemplate: any): Claim {
+  const claim = new Claim()
+  claim.response = Response.deserialize(responseTemplate)
+  return claim
+}
+
+function prepareDraft (draftTemplate: any): Draft<DraftClaimantResponse> {
+  return new Draft(0, 'claimantResponse', new DraftClaimantResponse().deserialize(draftTemplate), undefined, undefined)
+}
+
+function convertObjectLiteralToJSON (value: object): object {
+  return JSON.parse(JSON.stringify(value))
+}
+
+describe('CCJModelConverter - full admission by individual - claimant accepts', () => {
+  it('should convert to CCJ - defendant pays immediately', () => {
+    const claim = prepareClaim(fullAdmissionWithImmediatePaymentData)
+    const draft = prepareDraft(claimantResponseDraftWithPaymentMethodAccepted)
+
+    expect(convertObjectLiteralToJSON(CCJModelConverter.convertForIssue(claim, draft)))
+      .to.be.deep.equal(convertObjectLiteralToJSON(ccjIssueRequestPayImmediately))
+  })
+  it('should convert to CCJ - defendant pays by set date', () => {
+    const claim = prepareClaim(fullAdmissionWithPaymentBySetDateData)
+    const draft = prepareDraft(claimantResponseDraftWithPaymentMethodAccepted)
+
+    expect(convertObjectLiteralToJSON(CCJModelConverter.convertForIssue(claim, draft)))
+      .to.be.deep.equal(convertObjectLiteralToJSON(ccjIssueRequestPayBySetDate))
+  })
+  it('should convert CCJ - defendant pays by instalments', () => {
+    const claim = prepareClaim(fullAdmissionWithPaymentByInstalmentsData)
+    const draft = prepareDraft(claimantResponseDraftWithPaymentMethodAccepted)
+
+    expect(convertObjectLiteralToJSON(CCJModelConverter.convertForIssue(claim, draft)))
+      .to.be.deep.equal(convertObjectLiteralToJSON(ccjIssueRequestPayByInstalments))
+  })
+
+})
+
+describe('CCJModelConverter - full admission by individual - claimant rejects', () => {
+  xit('should convert to CCJ - defendant pays immediately - claimant propose new immediate payment', () => {
```

Comment:
I would recommend not committing skipped tests as they are easily forgotten about or missed,
if you need to commit or push with a failing test for review / help then you can add `--no-verify` to your commit and push commands

Reply:
Ok no problem, I was told to do so by Chris as the skipped tests cannot be tested at the moment, but wanted them in so they aren't forgotten about.

Reply:
Make sense, I will remove the tests for now